### DISCUSSION TITLE
Ability to edit files with basic ui support on mobile

### DISCUSSION THREAD

**Discussion by njradford** [community/community#3820] [Mobile] [answered]
It would be great to be able to edit files on mobile. If only in plain text for starters.

**Comment by nplasterer** [comment, marked as answer]
Hey thanks for your feedback! Feel free to upvote on this [topic related to code editing](https://github.com/github/feedback/discussions/3164) to help boost awareness of this request! - #3164
Also appreciate if you can share more details with us in the thread above 👆 - how would you like to edit code? Would you like to just edit files inside a pull request or would like to inside a repository?

**Reply by njradford** [reply]
@nplasterer - I think both a repo in general and PRs as well. it'd be useful to have an edit button present whenever a file is open. And on submit the user could be given a choice of "commit to master" or "create branch" as the Desktop UI does. I think at the moment I tend to only open the web UI on mobile if I need to make a small README change or fix a glaring bug.

**Comment by benhatsor** [comment]
A related project is [codeit.codes](https://codeit.codes), from which you can edit and run GitHub files on mobile.

**Comment by spaltrowitz** [comment]
This has now shipped! https://github.blog/changelog/2022-11-15-github-for-ios-edit-files-from-browse-code

**Reply by njradford** [reply]
@spaltrowitz - Awesome. Thank you for the notice!

**Reply by Mason-Lin** [reply]
The app got crashed when I tried to edit file on enterprise account.

Who should this issue be reported to and what information is required?
（Is the problem with the app or the server version is too old?）
